File: commands/fastres.lua
local vkapi = require "./api/vkapi"

local hello = {
    "А?";
    "М?";
    "Да да я";
    "Шо надо?";
    "Шо?";
    "Я тут"
}

return {
    keywords = {""};
    process = function(data)
        vkapi.message.send(data.peer_id, hello[math.random(#hello)])
    end
}

File: commands/fastres.js
const vk = require('../api/vk')

let answer = [
    'А?',
    'Да да я',
    'Я тут'
]

module.exports = {
    keywords: ['fastreshook'],
    callback: (data, args, cmd, bot) => {
        bot.setFastres(data.peer_id, true)
        vk.message.send(data.peer_id, answer[Math.floor(Math.random()*answer.length)])
    }
}